cmd/sync.go

```diff
@@ -127,7 +127,18 @@ func (raw *rawSyncCmdArgs) cook() (cookedSyncCmdArgs, error) {
 	// this if statement ladder remains instead of being separated to help determine valid combinations for sync
 	// consider making a map of valid source/dest combos and consolidating this to generic source/dest setups, akin to the lower if statement
 	// TODO: if expand the set of source/dest combos supported by sync, update this method the declarative test framework:
-	// TODO: add support for account-to-account operations (for those from-tos that support that)
+
+	/* We support DFS by using blob end-point of the account. We replace dfs by blob in src and dst */
+	if loc := inferArgumentLocation(raw.src); loc == common.ELocation.BlobFS() {
+		raw.src = strings.Replace(raw.src, ".dfs", ".blob", 1)
```

Comment:
Since we're choosing to "cleverly" change endpoints in sync, perhaps we should consider enabling such behavior for blob S2S as well?

Reply:
I'm skeptical even on this. @Ze had reasons to avoid copy. I'll reconfirm